src/nagendra/StringSumOfNumbers.java

```diff
@@ -0,0 +1,41 @@
+package nagendra;
+
+/*
+program 2 : sum of all numbers in a give string 
+input: te123ch9kj8 
+output : 123 + 9 + 8 = 140
+*/
+
+public class StringSumOfNumbers {
+
+	void getSumOfNumbers(String word) {
+		String number = "0";
+		int sum = 0;
+		for (int i = 0; i < word.length(); i++) {
+			if (Character.isDigit(word.charAt(i))) {
+				if (Character.isDigit(word.charAt(i)) && (i != word.length() - 1) && Character.isDigit(word.charAt(i + 1))) {
```

Comment:
Hi Nagendra, 
instead of putting this much condition, you can simplifies you code.
using single for loop you can handle this 
Please refer this code
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/752/files

Reply:
Hello Harsh,
I have updated the method and have tried to simplified the method and removed few loops.